modules/tinymce/src/plugins/table/main/ts/ui/Buttons.ts

```diff
@@ -30,164 +30,162 @@ const addButtons = (editor: Editor, selectionTargets: SelectionTargets): void =>
 
   const cmd = (command: string) => () => editor.execCommand(command);
 
-  // TODO TINY-8172: unwind this before merging the feature branch
-  const addButtonIfRegistered = (name: string, spec: AddButtonSpec) => {
-    if (editor.editorCommands.hasCustomCommand(spec.command)) {
-      editor.ui.registry.addButton(name, {
-        ...spec,
-        onAction: cmd(spec.command)
-      });
-    }
+  // TODO: TINY-8172 Look at solution to make sure user can't use buttons that won't do anything
+  const addButton = (name: string, spec: AddButtonSpec) => {
+    editor.ui.registry.addButton(name, {
+      ...spec,
+      onAction: cmd(spec.command)
+    });
```

Comment:
We won't have time to do TINY-8172 for the 6.0 release, so I'm afraid this needs to remain. I'm sorry if this wasn't communicated properly.

We need either this or TINY-8172 otherwise table features will appear to be available but have no effect in RTC (because any direct DOM mutations will be immediately reverted).